config/locales/en.yml

```diff
@@ -674,12 +674,16 @@ en:
         reason: Cancellation reasons
         rejection_category:
           fully_booked: The date you requested is fully booked
```

Comment:
Could we add a full stop at the end of this sentence?

Reply:
This has been fixed in #2177

Reply:
This has been fixed in #2177